.cirrus.yml

```diff
@@ -151,6 +167,8 @@ task:
         VERSION: 7-2
       matrix:
         - name: openbsd-vanilla
+          depends_on:
+           - cleanup-leftover-running
```

Comment:
Can't we do this depend outside of the matrix, instead of repeating it for each "arm"?

Reply:
If it is implemented like you said, 'depends_on at outside of matrix' overwrites open/netbsd-postgres task's depends_on. They starts to depend 'cleanup-leftover-running' instead of their vanilla tasks.